pizza_cutter/des_pizza_cutter/_constants.py

```diff
@@ -21,6 +21,8 @@
 BMASK_CUTOUT_EXTNAME = 'bmask_cutouts'
 ORMASK_CUTOUT_EXTNAME = 'ormask_cutouts'
 NOISE_CUTOUT_EXTNAME = 'noise_cutouts'
+INTERP_CUTOUT_EXTNAME = 'interp_cutouts'
+IMASK_CUTOUT_EXTNAME = 'imask_cutouts'
```

Comment:
@esheldon I am declaring new MEDS extensions `interp` and `imask`. They hold the value of any interpolated flux in the image and the fraction weighted by area of each pixel that is interpolated.

Reply:
Are you ok with these names?

Reply:
so `interp` means "interpolated flux" and `imask` means fraction of area of each pixel that is interpolated?  The correspondence of name and what they are seems a bit fuzzy to me.

Reply:
OK. Do you have suggestions? I tried and a few times and I got stuck. More than happy to rename!

Reply:
imask is the average of a 0 or 1 for SE image where 1 marks interpolated. So it is not quite the fraction of area of a pixel.

Reply:
maybe `interp_flux_cutouts` ?

I didn't follow "average of 0 or 1".   But maybe `fracmask_cutouts` ?  I don't know, I guess I don't know what the `i` in `imask` is standing for

Reply:
Ah. So suppose we have an image f_i of either 0 or 1 for each SE in the coadd and we have the weight w_i used for that image in the coadd. Then `imask` is `\sum f_i w_i / \sum w_i`. Does that make sense?

Reply:
What is `f_i` ?

but don't waste too much time on me, go ahead if you need to

Reply:
f_i is an array that has value 1 if a pixel is interpolated and zero otherwise

Reply:
OK.  As I said, go ahead without me I need to sign off for a while

Reply:
I refactored to something simpler.